base/install_iocs.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
I'd name this file `install_opcua.sh` instead.

Even if we have new modules which install IOCs, not every platform will support opcua specifically, and it's easier to have it as a single script which we can omit from the Dockerfile.

Reply:
When I talked to @henriquesimoes, I thought that the iocs folder would be deleted when I built other iocs. Keeping the iocs.sh script isolated for epics-in-doker native iocs, could facilitate other applications in the future?